test/kubemark/resources/manifests/addons/podsecuritypolicies/restricted.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: restricted
+  annotations:
+    kubernetes.io/description: 'restricted allows minimal access for create pods to work.'
+  labels:
+    addonmanager.kubernetes.io/mode: EnsureExists
+spec:
+  fsGroup:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+  - '*'
```

Comment:
Why this `restricted` not so strict as `cluster/addons/podsecuritypolicies/restricted.yaml`? I'd expect that they're the have similar restrictions (at this time they differ in `runAsUser` and `volumes` fields).